app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java

```diff
@@ -25,11 +30,38 @@
     private int selectedIndex;
 
     @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.settings_button, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int id = item.getItemId();
+
+        if (id == R.id.settings) {
+            Intent intent = new Intent(this, SettingsActivity.class);
+            startActivity(intent);
+            return true;
+        }
+
+        return super.onOptionsItemSelected(item);
+    }
+
+    @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.activity_main);
 
+        SharedPreferences settings = Preferences.get(getApplicationContext());
+        String getNotifications = Preferences.GET_NOTIFICATIONS;
+        if (!settings.contains(getNotifications)) {
```

Comment:
In `MainActivity` context this setting doesn't matter. Create method inside Preferences to get boolean if notificiations are enabled or not. `MainActivity` don't need to know about `Setting` implementation